### ISSUE TITLE
cPP/SD Correction

### ISSUE THREAD

**Issue by tmarconn** [ND-iTC/Documents#68]
**Provide the location of the issue**
Across the entire cPP and SD.

**What is the correction request for the cPP? Please describe.**
We believe that  "administrative user" should be replaced across both cPP and SD.

**Describe the solution you'd like**
Replace "administrative user" with "administrator" for consistency and clarity.

**Comment by tamcompton** [comment]
This seems like a reasonable and straightforward update

**Comment by plughy2** [comment]
I disagree with the proposed change.  I believe "administrative user" was purposely chosen for the requirements where it's found to allow for either the Administrator or an external IT entity to login.  Refer to the following portion of the Application Note under FIA_UIA_EXT. 1:

This requirement applies to users (Administrators and external IT entities) of services available from the TOE directly, and not services available by connecting through the TOE.

**Comment by tamcompton** [comment]
I don't think the term administrative user is any clearer for an IT entity than administrator. I still think the change can be made.

**Comment by dundiddat** [comment]
> 

@plughy2, the text you quoted from the application note under FIA_UIA_EXT.1 may actually be confusing the issue/terms.  It might be better to also reword that sentence to it instead says: "This requirement applies to Security Administrators and external IT entities of services available from the TOE directly, and not services available by connecting through the TOE."

**Comment by plughy2** [comment]
@dundiddat There's more than that single change.  Note the previously completed MINT task "Implementation of MINT task Local v remote authentication" updated the FIA_UIA_EXT.1 set of requirements. 

In reading FIA_UIA_EXT.1.2 and FIA_UIA_EXT.1.4, I don't see that it's possible to simply replace 'administrative user' with 'Security Administrator' mainly because 'Security Administrator' implies the user is already authorised and associated with the Security Administrator role, per FMT_SMR.2.  There would need to be additional analysis performed in this section to determine what else would need to be changed.

**Comment by dundiddat** [comment]
Posting this comment/request provided by Kirill:  "Please make sure that the final terminology is defined as NDcPP had issues in this area in the past. A while back NIT addressed this problem by creating a working definition- any role that has access to one or more of the explicitly defined management functions- but it is tied to the old terminology via application note."

**Comment by dundiddat** [comment]
Due to concerns expressed in the comments above, the MINT has decided to postpone resolution of this issue, so for now the status will be changed to "Backlog".
